SteamBot/SteamTradeDemoHandler.cs

```diff
@@ -202,9 +202,11 @@ public override void OnTradeSuccess()
         public override void OnTradeAwaitingEmailConfirmation(long tradeOfferID)
         {
             Log.Warn("Trade ended awaiting email confirmation");
-            SendChatMessage("Please complete the email confirmation to finish the trade");
+            SendChatMessage("Please complete the email confirmation to finish the trade, you've got 10 minutes from now.");
```

Comment:
Should the trade be cancelled? Polling stop after 10 minutes so the trade result becomes inconclusive.